README.md

```diff
@@ -45,7 +39,24 @@ project [aem-groovy-console](https://github.com/CID15/aem-groovy-console)
 
 2. Navigate to the [groovyconsole](http://localhost:4502/groovyconsole) page.
 
-#### Embedded package
+### Maven profiles
+
+Maven profiles can be used to install the bundles to AEM / Sling
+
+* AEM Author running on localhost:4502
+  * api & bundle: `-P autoInstallBundle`
+  * ui.apps, ui.apps.aem, ui.config, ui.content: `-P autoInstallPackage`
+  * all: `-P autoInstallSinglePackage`
+* AEM Publish running on localhost:4503
+  * api & bundle: `-P autoInstallBundle,publish`
```

Comment:
usually `autoInstallPackagePublish` is used as convention

Reply:
I know, but I hate the project archetype convention :( it forces you to duplicate everything. If you rather go for the convention, then I'll duplicate everything twice (once for publish and once for sling)

Reply:
+1